src/app.js

```diff
@@ -141,13 +148,36 @@ class Messenger extends EventEmitter {
       pageId: config.get('facebook.pageId')
     }));
 
-    this.app.get('/', (req, res) => {
-      res.send('👍');
-    });
+    this.app.post('/pause', bodyParser.json(), (req, res) => {
+      const { userId, paused } = req.body;
+      if (!userId && paused === undefined) {
+        res.sendStatus(400);
+        return;
+      }
 
-    this.app.get('/ping', (req, res) => {
-      res.send('pong');
+      const cacheKey = this.getCacheKey(userId);
+      this.cache.get(cacheKey)
+        .then((session/*: ?Session */)/*: Promise<Session> */ => {
+          if (!session) {
+            throw new Error("Can't pause unknown user");
```

Comment:
I didn't use a custom error like I did with `PausedUserError` because this code will fit on one screen, and the boilerplate for a custom error is a lot of lines I don't think will ever be useful.